addons/addon-base-raas-ui/packages/base-raas-ui/src/parts/accounts/AccountCard.js

```diff
@@ -233,7 +240,7 @@ class AccountCard extends React.Component {
   renderBudgetButton() {
     return (
       <Button floated="right" color="blue" onClick={this.handleBudgetButton}>
-        Budget Detail
+        {i18next.t('budgetDetail')}
```

Comment:
Suggest to move "budgetDetail" key into `main/solution/ui/public/locales/en/accounts.json` file

Reply:
done in the latest update